src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -191,6 +197,39 @@ public static Duration parseDuration(String duration) throws ParseException {
         }
     }
 
+    /**
+     * Parses a {@code String month} into a {@code Month}.
+     *
+     * @throws ParseException if the given {@code month} is not a valid month
+     */
+    public static Month parseMonth(String month) throws ParseException {
+        requireNonNull(month);
+        try {
+            return Month.of(Integer.parseInt(month.trim()));
+        } catch (DateTimeException | NumberFormatException e) {
+            throw new ParseException(MESSAGE_INVALID_MONTH);
+        }
+    }
+
+    /**
+     * Parses a {@code String year} into a {@code Year}.
+     *
+     * @throws ParseException if the given {@code year} is not a valid year.
+     */
+    public static Year parseYear(String year) throws ParseException {
+        requireNonNull(year);
+
+        try {
+            int yearValue = Integer.parseInt(year);
+            if (yearValue <= 0) {
```

Comment:
Just a small thing, would it make more sense if we set the condition to yearValue < 1970 or something?

Reply:
We can actually add a meeting for year 0001, we can sync up in v1.4 instead?